pressbooks/pressbooks-book
# Pb book 746 details

Remove a CSS rule which suppresses the expected show/hide caret for details summary elements. Fix for https://github.com/pressbooks/pressbooks-book/issues/746

To test:
1. Create a valid `<details>` element, like the one shown below: 
``` 
<details><summary><strong> What Are Accordions For?</strong><em>(show/hide)</em></summary>The H5P documentation spells it out nicely:
<blockquote>Reduce the amount of text presented to readers by using this responsive accordion. Readers decide which headlines to take a closer look at by expanding the title. Excellent for providing an overview with optional in-depth explanations.</blockquote>
You can see that this might be effective to reduce long lists of information, references, or breaking content into sections more like a miniature outline.
In this case, it’s used just a silly demo.
</details>
```
2. Observe that the summary element does not have the expected show/hide caret list style.

3. Apply this branch and reload the chapter. Observe that the show/hide caret element is visible for the summary element.
